pypi/data-processing/src/document_loaders/async_playwright.py

```diff
@@ -46,10 +46,17 @@ def __init__(
         Raises:
             ImportError: If the required 'playwright' package is not installed.
         """
-        self.url = url
-        self.max_count = max_count
-        self.max_depth = max_depth
-        self.interval_time = interval_time
+        if max_count is None:
+            max_count = 100
+        if max_depth is None:
+            max_depth = 1
+        if interval_time is None:
+            interval_time = 1000
+
+        self._url = url
+        self._max_count = max_count
+        self._max_depth = max_depth
+        self._interval_time = interval_time / 1000
```

Comment:
What is the interval_time is < 1000, will this interval_time converted to be  0 ???

Reply:
No, if interval_time=500, the value of interval_time will be converted to 0.5.